ci/run_tests_test.go

```diff
@@ -129,6 +130,7 @@ func TestRunTestsTimeout(t *testing.T) {
 func TestRecordResults(t *testing.T) {
 	db, cleanup := qtest.TestDB(t)
 	defer cleanup()
+	streamService := stream.NewStreamServices()
```

Comment:
Could you please add a test with a server streaming mocked data to a test client (in the same test func)?